skema/text_reading/text_reading/src/test/scala/org/ml4ai/skema/grounding/TestManualGrounder.scala

```diff
@@ -13,23 +13,24 @@ class TestManualGrounder extends Test {
 
   val manualGrounder: ManualGrounder = {
     val config = ConfigFactory.load().getConfig("Grounding")
-    val manualEntries = config.getString("manualGroundings")
+    val domainConfig = config.getConfig(config.getString("domain"))
+    val manualEntries = domainConfig.getString("manualGroundings")
 
     // Set alpha to 1.0 to bypass the edit distance algo for now
     ManualGrounder.fromFileOrResource(manualEntries, new FastNLPProcessor(withChunks = false, internStrings = false))
   }
 
-  "Manual grounder" should s"ground solar flare to the grounding concept with id tr:001" in {
+  "Manual grounder" should s"ground solar flare to the grounding concept with id tr:001" ignore {
```

Comment:
I don't know if it applies, but sometimes we've flagged failing tests with more information so that we come back to them.  They can be called failing with failing defined as ignore.  I can find examples if need be.